tests/candev/main.c

```diff
@@ -223,14 +223,20 @@ if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
 
     if (IS_ACTIVE(MCP2515_RECV_FILTER_EN)) {
         /* CAN filters examples */
-        struct can_filter filter[3];
+        struct can_filter filter[4];
         filter[0].can_mask = 0x7FF;
         filter[0].can_id = 0x001;   /* messages with CAN ID 0x001 will be received in mailbox 0 */
+        filter[0].target_mailbox = 0;
```

Comment:
Since this is only defined for `mpc2515`, CI build fails.

Reply:
Fixed